src/utilities/utility.ts

```diff
@@ -129,6 +131,93 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getFilePathsConfigs(): Promise<any> {

```

Comment:
Have an interface for the result. Instead of returning any

Reply:
Done